types/fetch-mock/index.d.ts

```diff
@@ -426,6 +434,14 @@ declare namespace fetchMock {
          */
         configure(opts: {}): void;
     }
+
+    interface FetchMockSandbox extends FetchMockStatic {
+        /**
+         * Also callable as fetch(). Use `typeof fetch` in your code to define
+         * a field that accepts both `fetch()` and a fetch-mock sandbox.
+         */
+        (input?: string | Request , init?: RequestInit): Promise<Response>;
```

Comment:
Can't say I've played with .sandbox() too much - does this always return a `Promise<Response>` or should this be the more generic `MockResponse` type from earlier in this file (which includes `Promise<Response>`)?

Reply:
I honestly have no idea, but since calling it is meant to be a drop-in replacement for `fetch()`, I doubt that you even want to treat this differently. Note that `MockResponse` is used for specifying mock responses. The callable here is meant to be used in actual code, that is obviously unaware of `fetch()` being mocked.

Reply:
That logic makes sense to me, and I can't see anything to contradict it in the source. This method on .sandbox() appears to come from https://github.com/wheresrhys/fetch-mock/blob/master/src/lib/fetch-handler.js#L5, which does look to return a `Promise<Response>`, so I'd be content to merge this - thank you for pulling it together!